common/djangoapps/third_party_auth/tests/test_provider.py

```diff
@@ -75,6 +78,23 @@ def test_enabled_returns_list_of_enabled_providers_sorted_by_name(self):
         with patch('third_party_auth.provider._PSA_OAUTH2_BACKENDS', backend_names):
             self.assertEqual(sorted(provider_names), [prov.name for prov in provider.Registry.enabled()])
 
+    def test_enabled_doesnt_query_site(self):
+        """Regression test for 1+N queries for django_site (ARCHBOM-1139)"""
+        re_django_site_query = re.compile(r'FROM\s+"django_site"')
+
+        self.enable_saml()
+        for i in range(100):
```

Comment:
Would 5 or 10 make the same point?

Reply:
I had set it as high as 100 so that there was room for slop in the test. But as long as that test down there checks for zero, I suppose this could be reduced to 5. I'll drop it to 5.